### DISCUSSION TITLE
Beta versions

### DISCUSSION THREAD

**Discussion by Cptbogo** [Nico640/docker-unms#82] [Q&A]
Just curious if you needed any assistance or what needs to be done to try some of the beta version of UNMS. I am interested in testing out at least 1.5.7.

Thank you, 
Ryan

**Comment by Nico640** [comment]
Hi,

the testing branch is now at 1.5.7. It's usually pretty straight forward to update, at least if there aren't any major changes. Most of the time I simply don't bother with beta versions because I personally don't use them and most people probably don't need them ;)
